src/prefect/environments/storage/docker.py

```diff
@@ -332,7 +332,8 @@ def create_dockerfile_object(self, directory: str = None) -> None:
                 ENV PREFECT__USER_CONFIG_PATH="/root/.prefect/config.toml"
                 {env_vars}
 
-                RUN pip install prefect
+                RUN pip install git+https://github.com/PrefectHQ/prefect.git@master#egg=prefect
```

Comment:
Should we add the `kubernetes` extras here?

Reply:
Yeah!